docs/package.json

```diff
@@ -3,13 +3,21 @@
 	"version": "0.0.0",
 	"private": true,
 	"scripts": {
+		"build": "robo build",
+		"deploy": "robo deploy",
+		"dev": "docusaurus serve --dir ./public/ && robox dev",
+		"doctor": "sage doctor",
+		"invite": "robo invite",
+		"lint:style": "prettier --write .",
+		"start": "docusaurus build --out-dir ./public/ && robo build && robo start",
+		"upgrade": "sage upgrade",
 		"docusaurus": "docusaurus",
-		"start": "docusaurus start",
-		"build": "docusaurus build",
+		"doco-start": "docusaurus start",
```

Comment:
I suggest merging commands instead to run in parallel. If not applicable, prefix with `:` instead.